### ISSUE TITLE
Baseline generation w/ type resolution produces UnusedImports false negative 

### ISSUE THREAD

**Issue by tyvsmith** [detekt/detekt#3123]
1.12.0 was fine for us.

When generating the baseline 1.13.1 and above flag every import as `UnusedImports`.
It does not flag them when running in standard mode. 

Potentially related to Issue #3079

**Comment by BraisGabin** [comment]
Can you elaborate a bit more? A snippet of code that reproduces the false positive would be nice.

**Comment by cortinico** [comment]
Also would be relevant to understand if you have Type Resolution enabled or not. We noticed several false positives in the past weeks related to `UnusedImports` with mixed behavior based on TR.

**Comment by tyvsmith** [comment]
Type resolution is enabled. It seems EVERY import statement is now flagged as unused in the baseline generation. In normal mode, it works as expected. I'll try to get a sample repro case this week.

**Comment by cortinico** [comment]
> Type resolution is enabled. It seems EVERY import statement is now flagged as unused in the baseline generation.

I believe we addressed this in https://github.com/detekt/detekt/pull/3079 that is included in `1.14.x`.
Have you tried bumping to that version (or is there a reason is holding you back on `1.13.1`)?

**Comment by tyvsmith** [comment]
> > Type resolution is enabled. It seems EVERY import statement is now flagged as unused in the baseline generation.
> 
> I believe we addressed this in #3079 that is included in `1.14.x`.
> Have you tried bumping to that version (or is there a reason is holding you back on `1.13.1`)?

The issue exists in all versions above 1.13.1 (including 1.14.0 and 1.14.1). 1.12.0 was the last working version. 

I also tried a build of the 1.14.1 tag with a revert on #3079 and it still occurred.

Note that we're using the CLI through a wrapper in Buck (no gradle plugin).

**Comment by tyvsmith** [comment]
We narrowed it down to a change in parsing CLI arguments. In previous versions (1.12 and below), we were able to delimit classpath jars using a semicolon. We were doing that using the detekt-cli through an args file.

```
--build-upon-default-config
--jvm-target
1.8
-c
config.yml
-i
Sample.kt
-cp
sample.jar;sample2.jar
-b
baseline.xml
-cb
```

In the newer versions, it seems a comma or colon is required.


```
--build-upon-default-config
--jvm-target
1.8
-c
config.yml
-i
Sample.kt
-cp
sample.jar:sample2.jar
-b
baseline.xml
-cb
```

The UnusedImport seemed to be a side effect, not exactly sure why that one was firing, but when resolving it in our args file generation, the problem dissapears. So going to mark this as closed.
